docs/reference/runtime/configuration.md

```diff
@@ -196,6 +196,8 @@ Allowing to configure the options in the agent as well as [interceptors](https:/
 
 ### `managementApi`
 
+> **Warning:** Experimental. The feature is not subject to semantic versioning rules. Non-backward compatible changes or removal may occur in any future release. Use of the feature is not recommended in production environments.
+
 An optional object that configures the Platformatic Management Api. If this object
 is not provided, the Platformatic Management Api will not be started. The options are
 passed directly to the Fastify server.
```

Comment:
```suggestion
passed directly to the Fastify server that backs the Management API. If enabled, it will listen to UNIX Socket/Windows named pipe located at `platformatic/pids/<PID>` inside the OS temporary folder.
```